app/components/WebsocketProvider.tsx

```diff
@@ -299,7 +299,11 @@ class WebsocketProvider extends React.Component<Props> {
         const collectionId = event.modelId;
         const deletedAt = new Date().toISOString();
 
-        const deletedDocuments = documents.inCollection(collectionId);
+        // drafts eventually are to be detached from collection rather than deleted, hence excluded here
+        const deletedDocuments = filter(
```

Comment:
Instead of filtering perhaps we should be clearing the `collectionId` in the `deletedDocuments.forEach` below? Won't drafts be left in the collection right now on clients?

Reply:
Technically, yes but it doesn't show up visually. Regardless, you're correct, it's better to clear `collectionId` for drafts here.